pages/docs/balance/business-verified-customer/handle-verification-statuses.mdx

```diff
@@ -780,21 +790,21 @@ If the document was found to be fraudulent or doesn’t match the identity of th
+
+| Failure reason            | Description                                                            | Detailed description                                                                                                                                                                                                                                                                                                 |
+| ------------------------- | ---------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `BusinessDocNotSupported` | Business document not supported                                        | The business document provided is not supported for verification. Please request approved [business documentation](#document-types) from the end user for verification.                                                                                                                                              |
+| `BusinessNameMismatch`    | Business name on account does not match document                       | The legal business name listed in the documentation uploaded does not match the Registered Business Name on the account. The account has been moved to [retry](#handling-status-retry) so that the business name listed at the account level can be adjusted to match the business documentation which was uploaded. |
+| `BusinessTypeMismatch`    | Business type chosen does not match document                           | Based on the documentation provided, the entity type for this business should be created as a/an (LLC, Corporation, Sole Prop). The account has been moved to [retry](#handling-status-retry) so that the correct business type (LLC, Corporation, Sole Prop etc.) can be submitted.                                 |
+| `ScanDobMismatch`         | Scan DOB does not match DOB on account                                 | The DOB listed on the ID uploaded does not match the DOB on the user's account. The account has been placed in [retry](#handling-status-retry) so that the user can adjust the DOB listed on the account to match the ID which was provided.                                                                         |
```

Comment:
```suggestion
| `ScanDobMismatch`         | Scan DOB does not match DOB on account                                 | The DOB listed on the ID uploaded does not match the DOB on the user's account. The account has been placed in [retry](#handling-retry-status) so that the user can adjust the DOB listed on the account to match the ID which was provided.                                                                         |
```